config/default.yml

```
@@ -899,6 +899,12 @@ Rails/ScopeArgs:
   Include:
     - app/models/**/*.rb
 
+Rails/SetIgnoredColumns:
+  Description: 'Looks for assignments of `ignored_columns` that override previous assignments.'
```

Can you add `StyleGuide` URL?

```suggestion
  Description: 'Looks for assignments of `ignored_columns` that override previous assignments.'
  StyleGuide: 'https://rails.rubystyle.guide/#append-ignored-columns'
```